src/main/java/marquez/service/DatasourceService.java

```diff
@@ -52,6 +54,16 @@ public Datasource create(@NonNull ConnectionUrl connectionUrl, @NonNull Datasour
     }
   }
 
+  public Optional<Datasource> get(@NonNull final UUID uuid) throws MarquezServiceException {
```

Comment:
For clarity, I recommend commenting either in code or the PR description on section of code that has changed between PRs when using stacked PRs.

Reply:
Yup - agreed. Still getting the hang of stacked PRs, and how to make things easier to review as changes occur.

Reply:
Next time I'll go back and apply the fixes and merge them back up to avoid confusion.